packages/ui/graphql/types/Report.ts

```diff
@@ -66,6 +75,49 @@ export const Report = objectType({
         );
       },
     });
+
+    t.nonNull.list.field('top', {
```

Comment:
`topSuggestions` is probably a better name

Reply:
added